doc/techsupport-manager/core_file_manager.md

```diff
@@ -0,0 +1,329 @@
+
+	root@sonic:/home/admin# coredumpctl list vlanmgrd
+	TIME PID UID GID SIG COREFILE EXE
+	Tue 2019-06-18 15:17:30 UTC 4458 0 0 6 present /usr/bin/vlanmgrd
+
+It is also possible to view detailed information along with backtrace of a particular core file using PID as search string.
+
+	root@sonic:/home/admin# coredumpctl dump 9275
+	PID: 9275 (hello)
+	UID: 0 (root)
+	GID: 0 (root)
+
+	Signal: 11 (SEGV)
+	Timestamp: Tue 2019-06-18 15:20:44 UTC (37min ago)
+	Command Line: ./hello
+	Executable: /home/admin/hello
+	Control Group: /system.slice/system-serial\x2dgetty.slice/serial-getty@ttyS2.service
+	Unit: serial-getty@ttyS2.service
+	Slice: system-serial\x2dgetty.slice
+	Boot ID: f687508200b948a58c20222631c5ebbc
+	Machine ID: 7bfb28a79813456090ec1df35a2073f5
+	Hostname: sonic
+
+	Storage: /var/lib/systemd/coredump/core.hello.0.f687508200b948a58c20222631c5ebbc.9275.1560871244000000000000.lz4
+	Message: Process 9275 (hello) of user 0 dumped core.
+
+	Stack trace of thread 9275:
+	#0 0x00000000004005e7 main (hello)
+	#1 0x00007f6ece5712e1 __libc_start_main (libc.so.6)
+	#2 0x0000000000400519 _start (hello)
+	Refusing to dump core to tty (use shell redirection or specify --output).
+
+  
+## Core File Rotation and Archive
+
+When core file is generated for the same process multiple times, the framework should keep the generated core file in compressed form for the last N-1 number of core files. The number and the maximum size of the core files can be configured through /etc/systemd/coredump.conf file. The latest core file shall be kept in uncompressed format.
+
+The archived core file is generated in a pre-defined format by the systemd-coredump tool.
+
+Format: core.<program-name>.<uid>.<boot-id>.<PID>.<timestamp>
+Examples:
+	core.orchagent.0.8bc64adf67544e9e8b897cc5c1c9fde7.10618.1479890855000000000000.lz4  
+	core.orchagent.0.8bc64adf67544e9e8b897cc5c1c9fde7.11686.1479886973000000000000.lz4  
+	core.orchagent.0.8bc64adf67544e9e8b897cc5c1c9fde7.1748.1479887528000000000000.lz4  
+	core.orchagent.0.8bc64adf67544e9e8b897cc5c1c9fde7.18600.1479888638000000000000.lz4  
+	core.orchagent.0.8bc64adf67544e9e8b897cc5c1c9fde7.21734.1479888083000000000000.lz4  
+	core.orchagent.0.8bc64adf67544e9e8b897cc5c1c9fde7.23065.1479890300000000000000.lz4  
+	core.orchagent.0.8bc64adf67544e9e8b897cc5c1c9fde7.26069.1479889746000000000000.lz4  
+	core.orchagent.0.8bc64adf67544e9e8b897cc5c1c9fde7.31104.1479891410000000000000.lz4  
+	core.orchagent.0.8bc64adf67544e9e8b897cc5c1c9fde7.5952.1479889193000000000000.lz4
+  
+# Tech-support export service 
+
+The tech-support data is a vital information for debugging of a system and is captured by collecting the  device configuration, system information, log files  and core files. The export service captures the tech-support data and export it to a remote server for better offline debugging.  The tech-support data is captured and exported under the following conditions 
```

Comment:
Is this part of systemd-coredump service?
Will we be able to provide custom script to generate tech-support data?

Reply:
Export service is separate service.  The standard "generate_dump" script is used for generating the techsupport data